src/modules/reading-room/types/metadata.ts

```diff
@@ -0,0 +1,17 @@
+// 2-letter for url parsing
+export enum MetadataProp {
+	CreatedAt = 'aa', // creatiedatum
+	Creator = 'ab', // maker
+	Description = 'ac', // beschrijving
+	Duration = 'ad', // duurtijd
+	Era = 'ae', // tijdsperiode
+	Everything = 'af', // alles
+	Genre = 'ag', // genre
+	Language = 'ah', // taal
+	Location = 'ai', // locatie
+	Mediatype = 'aj', // bestandstype
+	Medium = 'ak', // drager
+	PublishedAt = 'al', // uitzenddatum
+	Publisher = 'am', // publisher
+	Title = 'an', // titel
```

Comment:
I would leave this as json, just since it has a lower chance of causing parsing issues down the line. But we can leave it as is now, and fix issues or switch back to json if we encounter any problems.

Reply:
`src/modules/reading-room/const/query-params.ts` makes this approach as secure as a default `JsonParam` but doesn't mangle the URL.